### PR TITLE
Attempt to add new/missing websocket message

### PR DESCRIPTION
**Pull request by Ravendwyr**
### Checklist

- [x] Tests have been added in relevant areas
- [x] Corresponding changes made to the documentation (README.adoc) 

### Type of change

"Bug fix"

## Description

This PR adds support for a `read-all-notifications` websocket message that is sent/received when clicking the envelope icon in the notifications menu to mark any unread notifications as read. Previously this would cause an exception for an unhandled message type. This change at the very least suppresses this.

### CODE REVIEW

**Comment by Rakambda** [comment]
Thanks.
Is it possible to add a test? You can basically copy paste https://github.com/RakambdaOrg/ChannelPointsMiner/blob/a05c38b867a16265b22ae5d1b4d9fe5798e56928/miner/src/test/java/fr/rakambda/channelpointsminer/miner/api/ws/TwitchPubSubWebSocketClientMessageTest.java#L772 and adapt set the new message type in the payload.

**Comment by Ravendwyr** [comment]
Test added. Sorry about that, I missed that! I think I did it right?

File: miner/src/test/java/fr/rakambda/channelpointsminer/miner/api/ws/TwitchPubSubWebSocketClientMessageTest.java [hunk 1]
```diff
@@ -817,6 +817,54 @@ void onReadNotifications(WebsocketMockServer server){
...
+		server.send(TestUtils.getAllResourceContent("api/ws/readAllNotifications.json"));
+
+		var expected = MessageResponse.builder()
+				.data(MessageData.builder()
+						.topic(Topic.builder()
+								.name(ONSITE_NOTIFICATIONS)
+								.target("123456789")
+								.build())
+						.message(ReadNotifications.builder()
+								.data(ReadNotificationsData.builder()
```
Comment (Rakambda) [inline miner/src/test/java/fr/rakambda/channelpointsminer/miner/api/ws/TwitchPubSubWebSocketClientMessageTest.java:831]:
```suggestion
						.message(ReadAllNotifications.builder()
								.data(ReadAllNotificationsData.builder()
```

**Review by Rakambda — CHANGES_REQUESTED** [review]
Almost ;)

**Review by Rakambda — APPROVED** [review]

File: miner/src/test/java/fr/rakambda/channelpointsminer/miner/api/ws/TwitchPubSubWebSocketClientMessageTest.java (hunk 1 above)
Comment (Ravendwyr) [reply miner/src/test/java/fr/rakambda/channelpointsminer/miner/api/ws/TwitchPubSubWebSocketClientMessageTest.java]: Oops 😓

**Pull request merged**